doc/classes/Control.xml

```diff
@@ -727,7 +759,7 @@
 			<argument index="0" name="name" type="StringName">
 			</argument>
 			<description>
-				Removes a theme override for a [Color] with the given [code]name[/code].
+				Removes a local override for a theme [Color] with the specified [code]name[/code] previously added by [method add_theme_color_override].
```

Comment:
```suggestion
				Removes a local override for a theme [Color] with the specified [code]name[/code].
				See also [method add_theme_color_override].
```
It's probably fine as is, but it's also possible to add overrides from the editor instead of from a script (although I assume `add_theme_color_override()` is still getting called behind the scenes).
Same thing for the other `remove_theme_x_override()` methods.

Reply:
The mention is there more for the purposes of cross-linking than for actually adding some information, but I can amend it to read "or via the Inspector dock", sure.